.github/workflows/Build.yaml

```diff
@@ -125,6 +125,12 @@ jobs:
           name: lint-reports
           path: '**/build/reports/lint-results-*.html'
 
+      - name: Upload lint reports (SARIF)
+        if: always()
```

Comment:
One case to consider is when the build fails, this task will also run. However, it will be unable to find the file and throw an error at a later stage.

Reply:
Not necessarily, it really depends when/where the build fails.
That being said, this could be done globally on another PR (not only for lint related tasks).